content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -583,6 +585,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
   [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
 - `LegacyServiceAccountTokenTracking`: Track usage of Secret-based
   [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
+- `LegacyServiceAccountTokenCleanUp`: Clean up Secret-based
+  [service account tokens](/docs/reference/access-authn-authz/authentication/#service-account-tokens).
```

Comment:
We try to keep this sorted, so how about:
```suggestion
- `LegacyServiceAccountTokenCleanUp`: Clean up legacy Secret-based
  [service account tokens](/docs/concepts/security/service-accounts/#get-a-token).
- `LegacyServiceAccountTokenTracking`: Track usage of legacy Secret-based
  [service account tokens](/docs/concepts/security/service-accounts/#get-a-token).
```

I would also link to https://k8s.io/docs/concepts/security/service-accounts/#get-a-token; that concept page didn't exist when the TokenRequest work began.